tests/langchain/sample_code/chain.py

```diff
@@ -1,3 +1,7 @@
+import dbutils
+
+dbutils.library.restartPython()
```

Comment:
also tested without import, added because of formatting error as follows
`tests/langchain/sample_code/chain.py:1:1: F821 Undefined name `dbutils``